mk/k3d.mk

```diff
@@ -1,20 +1,3 @@
-EXAMPLE_NAMESPACE ?= kuma-example
-KIND_KUBECONFIG_DIR ?= $(HOME)/.kube
-KIND_KUBECONFIG ?= $(KIND_KUBECONFIG_DIR)/kind-kuma-config
-KIND_CLUSTER_NAME ?= kuma
-
-ifdef IPV6
-KIND_CONFIG ?= $(TOP)/test/kind/cluster-ipv6.yaml
-else
-KIND_CONFIG ?= $(TOP)/test/kind/cluster.yaml
-endif
```

Comment:
I believe these are needed, are these expected in some other file?

Reply:
Moth `kind.mk` and `k3d.mk` use these definitions, but we don't need to duplicate them in both files AFAIK. If we get to the point where we need to re-order the includes, we can hoist this out into some sort of common defs include file.

Reply:
@nickolaev ping!